src/windows/mainwindow.hpp

```diff
@@ -3,6 +3,7 @@
 #include <vector>
 #include "src/widgets/navbar.hpp"
 #include "src/views/view.hpp"
+#include "src/widgets/navbar.hpp"
```

Comment:
Woah... trying to be sneaky?

Reply:
Gotta keep you sharp, you passed the test

Reply:
I knew you would pull something like this GOTCHA!